docs/CodingStandards

```diff
@@ -15,30 +15,26 @@ gg=G
 In emacs:
 ? (C-x C-c, vi filename?)
 
-PHP: 
-- for control structures, opening curly bracket should be on the same line as statement with a space before the curly bracket, closing bracket should be aligned with opening of statement. 
-    if (x) {
-        things go here;
-    }
-- An exception to this is class definitions and class function definitions, which may have the bracket on the proceeding line:
-
-class NDB_BVL_Foo extends NDB_Bar 
-{
-    definition goes here
-}
-
-Instead of using PEAR::isError, use Utility::isErrorX. It does the same thing, but avoids errors about PEAR::isError not being static in newer versions of PHP.
+PHP:
+- Ensure formatting meets Loris style guidelines by running phpcs with the LorisCS.xml
+  configuration file which accompanies these guidelines. You can run the tool
+  with the command `vendor/bin/phpcs --standard=docs/LorisCS.xml [file]` for any
+  files you've modified or added. For new modules, ensure that PHPCS has been
+  run on the module directory and add the module to travis.yml
 
 HTML:
-- HTML should never be mixed with code. HTML should go into a template and be rendered using a templating library (smarty for PHP)
-- General formatting rules about indentation applies for each tag embedded inside another tag of HTML
+- HTML should never be mixed with code. HTML should go into a template and be
+  rendered using a templating library (smarty for PHP)
+- General formatting rules about indentation applies for each tag embedded inside
+  another tag of HTML
 <div>
     <span>foo</span>
 </div>
 
 Javascript: 
-- Javascript should never be mixed with HTML or PHP code. Javascript should go into htdocs/JS/modules/ (Loris automatically includes the js file in that directory with the same name as Test_name, if it exists as of https://github.com/aces/Loris-Trunk/commit/8065fb265705298cb801d0e2f373d75e51d3cf29)
-- Any newly written Javascript should pass JSLint with default options, since those enforce the coding standards adopted by the general Javascript community
+- Javascript should never be mixed with HTML or PHP code. Javascript should go into
+  `modules/js`
```

Comment:
@driusan Isn't the path for javascript `/modules/$module_name$/js` . Also, maybe we should mention something about react. (i.e all react javascript goes into `/modules/$module_name$/jsx)